collect_app/lint.xml

```diff
@@ -1,3 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <lint>
+    <!-- make sure classes referenced in the manifest or in a layout file are in the project -->
+    <issue id="MissingRegistered" severity="error" />
+
+    <!-- require inputType attribute on all text fields -->
+    <issue id="TextFields" severity="error" />
```

Comment:
I skimmed through some of the warnings and picked out some that were easy fixes (few existing failures) and marked as higher priority by lint. Having the right keyboard is important for usability so I thought it was a good one to enforce. I'm not super committed to it if you'd rather bring it back down to warning.